File: LDC.Atlas.WebApp/Client/app/shared/entities/physical-trade-bulk-edit.ts
import { PhysicalContractToUpdate } from './physical-contract-to-update.entity';
import { SectionToUpdate } from './section-to-update.entity';

export class PhysicalTradeBulkEdit {
    companyId: string;
    physicalContractToUpdate: PhysicalContractToUpdate[];
    sectionToUpdate: SectionToUpdate[];

    constructor(companyId: string, physicalContractToUpdate: PhysicalContractToUpdate[] = null,
        sectionToUpdate: SectionToUpdate[] = null) {
        this.companyId = companyId;
        this.physicalContractToUpdate = physicalContractToUpdate;
        this.sectionToUpdate = sectionToUpdate;
    }
}

File: LDC.Atlas.Services.Trading/Entities/PhysicalTradeBulkEdit.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace LDC.Atlas.Services.Trading.Entities
{
    public class PhysicalTradeBulkEdit
    {
        internal string CompanyId { get; set; }

        public long? DataVersionId { get; set; }


        public IEnumerable<PhysicalContractToUpdate> PhysicalContractToUpdate { get; set; }

        public IEnumerable<SectionToUpdate> SectionToUpdate { get; set; }
    }
}
